src/docbuilder/rustwide_builder.rs

```diff
@@ -12,6 +12,7 @@ use crate::utils::{copy_dir_all, parse_rustc_version, CargoMetadata};
 use crate::{db::blacklist::is_blacklisted, utils::MetadataPackage};
 use crate::{Config, Context, Index, Metrics, Storage};
 use docsrs_metadata::{Metadata, DEFAULT_TARGETS, HOST_TARGET};
+#[cfg(target_os = "linux")]
 use failure::ResultExt;
```

Comment:
this is only used below in a section that is only compiled on linux, so this `use` is unused on all other platforms

Reply:
Could you move it into the function that uses it instead? That way the cfgs are automatically in sync instead of needing to update them in two places.

Reply:
I moved it into the block that uses it, not sure if that is the most idiomatic way :) 

( I can move it to the function if that's what is better)

Reply:
Yup, that's fine :) thanks!